docs/visualizing-your-sql-queries/plotting-with-plotly.md

```diff
@@ -1,14 +1,409 @@
 ---
-jupyter:
-  jupytext:
-    text_representation:
-      extension: .md
-      format_name: markdown
-      format_version: '1.3'
-      jupytext_version: 1.14.5
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.6
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
 ---
 
 # Plotting with `plotly`
 
+Plotly is a visualization library specifically designed for dynamic interactive plots. The library offers several additional effects to visualizations, such as zooming, panning, and hovering effects. This library is especially known for being easily deployed with web applications.
 
+For more on plotly, visit: https://plotly.com/python/
 
+Let's see how we can apply plotly to our familiar bank marketing data sets.
+
+## Install - execute this once.
+
+```{important}
+<b>Note:</b> The `--save` and `%sqlcmd` features used require the latest JupySQL version. Ensure you run the code below to update JupySQL.
+```
+
+This code installs JupySQL, DuckDB, and Pandas in your environment. We will be using these moving forward.
+
+```{code-cell} ipython3
+%pip install jupysql --upgrade duckdb-engine pandas plotly --quiet
+```
+
+## Load the data
+
+```{important}
+<b>Note:</b> If you are following these lessons locally and <b>not</b> on Google Colab, then there is no need to load the data again.
+```
+
+This section was covered in detail in the previous tutorial: [Joining Data in SQL](https://ploomber-sql.readthedocs.io/en/latest/intro-to-sql/joining-data-in-sql.html#load-the-data). We will be using the same data in this tutorial as well.
+
+```{code-cell} ipython3
+import sys
+import plotly.express as px
+
+sys.path.insert(0, "../../")
+import banking  # noqa: E402
```

Comment:
Where are we getting this package from? I think I saw this in the other PR as well and the notebook kept failing with `banking module not found` error

Reply:
This script is located in the root folder. @lfunderburk made it so that this script would exist solely at the root level so that different modules can access it.

I don't get the `banking module not found` error when I build the doc. Can you tell me which PR/notebook is giving you this error?

Reply:
[This PR](https://github.com/ploomber/sql/pull/56/files#diff-8c094911daccb5249ab98d988fa1b04779e193ab65260eb3b2dea70c3e0d5f5e).

I got the error when running the notebook in Google colab. It's not clear from the tutorial alone as to where the root folder is. And I think, ideally the user should be able to run each tutorial as is in their own environment/ colab without having to run any extra steps.

Reply:
The Google colabs are something I am grappling with in general. 
Google colab doesn't seem to read the full repo, just the directory pointing at the notebook
I wanted to minimize having copies of the same .py script in multiple folders.
Will create a separate issue to fix the colab component. marking this as resolved